packages/matchbox/src/components/Modal/Modal.js

```diff
@@ -65,7 +65,7 @@ class Modal extends Component {
     );
 
     return (
-      <div className={modalClasses} onClose={onClose} {...rest} ref={(node) => this.container = node}>
+      <div className={modalClasses} onClose={onClose} {...rest} ref={(node) => this.container = node} role="dialog" aria-modal="true">
         <Grid center='xs' middle='xs' className={styles.Grid}>
           <Grid.Column xs={11} md={9} xl={7}>
             <Content contentRef={(node) => this.content = node} open={open}>
```

Comment:
Was wondering if we'd need `aria-hidden=true` or something similar when the modal is not open, since the modal containers are always mounted through a portal. But it looks like RTG handles mounting/unmounting children correctly for us so there is never any content rendered when modals are closed